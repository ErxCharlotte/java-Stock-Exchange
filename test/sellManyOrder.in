ADD trader1 100.00
ADD trader2 100.00
ADD trader3 100.00
ADD trader4 100.00
ADD trader5 100.00
IMPORT trader1 ABC 100
IMPORT trader2 ABC 100
IMPORT trader3 ABC 100
IMPORT trader4 ABC 100
SELL trader1 ABC 10 50
AMOUNT trader1 ABC
SELL trader1 ABC 10 500
AMOUNT trader1 ABC
SELL trader2 ABC 10 100
SELL trader3 ABC 10 200
SELL trader4 ABC 10 10
SELL trader3 ABC 10 10
SELL trader2 ABC 10 400
AMOUNT trader1 ABC
BUY trader5 ABC 100 300
AMOUNT trader1 ABC
TRADES
AMOUNT trader1 ABC
AMOUNT trader2 ABC
AMOUNT trader3 ABC
AMOUNT trader4 ABC
AMOUNT trader5 ABC
BALANCE trader1
BALANCE trader2
BALANCE trader3
BALANCE trader4
BALANCE trader5
BOOK BUY
BOOK SELL
EXIT