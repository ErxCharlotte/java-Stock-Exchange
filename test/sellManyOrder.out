$ Success.
$ Success.
$ Success.
$ Success.
$ Success.
$ Trader now has 100.00 units of ABC.
$ Trader now has 100.00 units of ABC.
$ Trader now has 100.00 units of ABC.
$ Trader now has 100.00 units of ABC.
$ No trades could be made, order added to sell book.
$ 90.00
$ No trades could be made, order added to sell book.
$ 80.00
$ No trades could be made, order added to sell book.
$ No trades could be made, order added to sell book.
$ No trades could be made, order added to sell book.
$ No trades could be made, order added to sell book.
$ No trades could be made, order added to sell book.
$ 80.00
$ Product bought in part, trades as follows:
trader4->trader5: 10.00xABC for $10.00.
trader3->trader5: 10.00xABC for $10.00.
trader1->trader5: 10.00xABC for $50.00.
trader2->trader5: 10.00xABC for $100.00.
trader3->trader5: 10.00xABC for $200.00.
$ 80.00
$ trader4->trader5: 10.00xABC for $10.00.
trader3->trader5: 10.00xABC for $10.00.
trader1->trader5: 10.00xABC for $50.00.
trader2->trader5: 10.00xABC for $100.00.
trader3->trader5: 10.00xABC for $200.00.
$ 80.00
$ 80.00
$ 80.00
$ 90.00
$ 50.00
$ $600.00
$ $1100.00
$ $2200.00
$ $200.00
$ $-3600.00
$ 0007: BUY 50.00xABC @ $300.00
$ 0001: SELL 10.00xABC @ $500.00
0006: SELL 10.00xABC @ $400.00
$ Have a nice day.